From 86c3fe1d857ecffa9c764ff44f140c19a3d7510e Mon Sep 17 00:00:00 2001
From: Allen Luce <allen.luce@prometheanworld.com>
Date: Fri, 29 Oct 2021 01:05:12 +0000
Subject: [PATCH] Force tablet mode

This ensures that the toolbar will have back/forward/reload buttons
and generally gives a better experience on panels.
---
 components/browser_ui/accessibility/android/java/src/org/chromium/components/browser_ui/accessibility/AccessibilitySettings.java   | 2 +-
 ui/android/java/src/org/chromium/ui/base/DeviceFormFactor.java  | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/components/browser_ui/accessibility/android/java/src/org/chromium/components/browser_ui/accessibility/AccessibilitySettings.java b/components/browser_ui/accessibility/android/java/src/org/chromium/components/browser_ui/accessibility/AccessibilitySettings.java
--- a/components/browser_ui/accessibility/android/java/src/org/chromium/components/browser_ui/accessibility/AccessibilitySettings.java
+++ b/components/browser_ui/accessibility/android/java/src/org/chromium/components/browser_ui/accessibility/AccessibilitySettings.java
@@ -99,7 +99,7 @@ public class AccessibilitySettings
         ChromeBaseCheckBoxPreference forceTabletUiPref =
                 (ChromeBaseCheckBoxPreference) findPreference(PREF_FORCE_TABLET_UI);
         forceTabletUiPref.setChecked(SharedPreferencesManager.getInstance().readBoolean(
-                            ChromePreferenceKeys.FLAGS_FORCE_TABLET_UI_ENABLED, false));
+                            ChromePreferenceKeys.FLAGS_FORCE_TABLET_UI_ENABLED, true));
         forceTabletUiPref.setOnPreferenceChangeListener(this);
 
         ChromeBaseCheckBoxPreference accessibilityTabSwitcherPref =
diff --git a/ui/android/java/src/org/chromium/ui/base/DeviceFormFactor.java b/ui/android/java/src/org/chromium/ui/base/DeviceFormFactor.java
--- a/ui/android/java/src/org/chromium/ui/base/DeviceFormFactor.java
+++ b/ui/android/java/src/org/chromium/ui/base/DeviceFormFactor.java
@@ -63,7 +63,7 @@ public class DeviceFormFactor {
      */
     public static boolean isNonMultiDisplayContextOnTablet(Context context) {
         if (SharedPreferencesManager.getInstance().readBoolean(
-                ChromePreferenceKeys.FLAGS_FORCE_TABLET_UI_ENABLED, false))
+                ChromePreferenceKeys.FLAGS_FORCE_TABLET_UI_ENABLED, true))
             return true;
         return detectScreenWidthBucket(context) >= SCREEN_BUCKET_TABLET;
     }
-- 
2.17.1

